bindings/SofaRuntime/CMakeLists.txt

```diff
@@ -26,7 +25,7 @@ SP3_add_python_module(
         DESTINATION  SofaRuntime
         SOURCES      ${SOURCE_FILES}
         HEADERS      ${HEADER_FILES}
-        DEPENDS      Sofa.Core Sofa.Helper Sofa.SimulationCore SofaSimulationGraph SofaSimulationCommon SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
+        DEPENDS      Sofa.Simulation.Graph SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
```

Comment:
Should be propagated to [Bindings.SofaRuntimeConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/SofaRuntime/Bindings.SofaRuntimeConfig.cmake.in)